### PR TITLE
Added in view all reviews view

### PR DESCRIPTION
**Pull request by omris-elda**

### CODE REVIEW

File: src/components/Reviews/Reviews.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import ReviewCard from "../ReviewCard/ReviewCard";
+
+export default function Reviews() {
+
+    const [loading, setLoading] = useState(true);
+    const [reviewsList, setReviewsList] = useState([]);
+
+    useEffect(() => {
+        setLoading(true);
+        console.log("inside of the reviews useeffect");
```
Comment (dappernerddesigns) [inline src/components/Reviews/Reviews.jsx]: Make sure to remove console logs when you don't need them anymore

**Review by dappernerddesigns — CHANGES_REQUESTED** [review]
There is very little contrast between your background colour and your text. This makes it incredibly hard to read and may cause eye strain.

This PR does not list all reviews in browser because you have not provided a link to them. You have rendered your review cards on the path of /reviews but not provided any navigation to that path.

When going to that path, the review cards should not be showing the body of the review. For a longer review that would mean a huge card! The ticket for this in BE would have stated that the body key was not in the list of what we were looking for. 

On a mobile screen your images are made so small that they may as well not be there. Consider how the card lays out on a mobile screen as that is most likely the device being used to view this application. 

Make the changes suggested and please resubmit this PR.

**Comment by omris-elda** [comment]
The body is still in there, however now it will not exceed 150 characters, keeping the cards nice and short. Since the body is already in the back end, it seems a waste not to use it and the cards look bland and boring otherwise, with no inspiration to click on them.

**Review by k-huggs — COMMENTED** [review]
hi mate your queries shouldn't be in your components folder.
All good to merge!

**Pull request merged**